src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -221,7 +227,10 @@ protected override string GenerateCommandLineCommands ()
 
 		protected override string GenerateFullPathToTool ()
 		{
-			return Path.Combine (ToolPath, ToolExe);
+			var fullToolPath = Path.Combine (ToolPath, ToolExe);;
+			if (File.Exists (fullToolPath))
+				return fullToolPath;
+			return Path.Combine (ToolPath, "bin", ToolExe);
```

Comment:
This method seems weirdly duplicative of what's being added to `Execute()`. Is that duplication necessary?